IDEAS/Examples/Tutorial/CrashCourse/Exercise3/Example1.mo

```diff
@@ -76,31 +75,31 @@ model Example1 "First example model containing one zone"
         origin={70,10})));
 equation
   connect(outerWall.propsBus_a, zone.propsBus[1]) annotation (Line(
-      points={{-45,12},{-38.5,12},{-38.5,15.7143},{-20,15.7143}},
+      points={{-45,12},{-38.5,12},{-38.5,13.1429},{-20,13.1429}},
```

Comment:
I see quite some unintended changes, due to different Dymola versions. Sometimes Dymola changes other lines in the text if you save a model. However, it is a good practice to check the lines of code that you commit and only commit the lines that you really want to change. You can discard all other changes. 

As there are many, and you will change the structure anyway, I propose to start from a new branch, which is easier than reverting these changes.